RevenueCatUI/CustomerCenter/Views/CustomerCenterView.swift

```diff
@@ -157,10 +157,24 @@ private extension CustomerCenterView {
                 }
             }
         }
+        .manageSubscriptionsSheet(isPresented: $viewModel.manageSubscriptionsSheet)
         .modifier(CustomerCenterActionViewModifier(actionWrapper: viewModel.actionWrapper))
+        .onCustomerCenterFeedbackSurveyCompleted { _ in
+            viewModel.manageSubscriptionsSheet = true
+        }
         .onCustomerCenterPromotionalOfferSuccess {
             Task {
-                await viewModel.loadScreen()
+                await viewModel.loadScreen(shouldSync: true)
+            }
+        }
+        .onCustomerCenterShowingManageSubscriptions {
```

Comment:
If developers set `onCustomerCenterShowingManageSubscriptions` themselves on `CustomerCenterView`, will they also get their modifer executed? Or does setting the modifier here "swallow " the modifier. I think that's what you are testing in `testNestedActionWrappers` right? but that's only testing `onCustomerCenterPromotionalOfferSuccess`

Reply:
The test I've added covers that case 🚀